saas/fsm/renewal_reminder/README.md

```diff
@@ -19,11 +19,21 @@ This policy has the following input parameters required when launching the polic
 - *Upcoming Number of Days* - If an application renewal is due in the upcoming time period, raise an incident
 - *Email addresses to notify* - Email addresses of the recipients you wish to notify
 
-## Policy Actions
+## Prerequisites
 
-- Sends an email notification
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+In the log analytics workspace provided, Microsoft-Windows-Disk/Operational event log should be enabled.
```

Comment:
Looks like it was carried over from another policy

Reply:
Sorry,now its updated.